run_unittests.py

```diff
@@ -381,6 +381,8 @@ def _run(self, command, workdir=None):
         output = p.communicate()[0]
         print(output)
         if p.returncode != 0:
+            if 'MESON_SKIP_TEST' in output:
+                raise unittest.SkipTest('Project requested skipping.')
```

Comment:
Nitpick: "Project test case" instead?

Also this doesn't tell the reader what test case caused the skip.